includes/apple-exporter/components/class-podcast.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * Publish to Apple News Includes: Apple_Exporter\Components\Podcast class
+ *
+ * Contains a class which is used to transform Podcast embeds into Apple News format.
+ * Documentation about apple podcast players can be found here: https://podcasters.apple.com/support/889-apple-podcasts-embed-player.
+ *
+ * @package Apple_News
+ * @subpackage Apple_Exporter
+ * @since 0.2.0
```

Comment:
This should be updated to the version of the plugin where it was introduced, which in this case is `2.4.0`